### PR TITLE
Added SVR Loss

### PR DESCRIPTION
**Pull request by czgdp1807**
### Summary
I have added the SVR loss function. Though hinge loss is already there, however, the definition of hinge loss is different from the one that I have added. The one which I have added is called, e-insensitive loss function. I believe Keras should also have `support_vector_regression_loss`.

e should be interpreted as epsilon. 

### References
..[1] http://kernelsvm.tripod.com/

- [x] This PR requires new unit tests [y/n] (make sure tests are included)
- [x] This PR requires to update the documentation [y/n] (make sure the docs are up-to-date)
- [x] This PR is backwards compatible [y/n]
- [ ] This PR changes the current API [y/n] (all API changes need to be approved by fchollet)

### CODE REVIEW

**Comment by czgdp1807** [comment]
I am a beginner in contributing to keras. Please guide me and correct me if I have done any mistake.

**Comment by fchollet** [comment]
Thank you for the PR.

The loss function added is defined as `K.maximum(0., K.abs(y_true - y_pred) - 0.1)`.

- If the difference between `y_true` and `y_pred` is greater than 0.1 (e.g. at the beginning of training), this loss is equivalent to `MAE`.
- If the difference between `y_true` and `y_pred` is smaller than 0.1 (e.g. after training for a while), this loss is no longer differentiable and training will stall.

As such, this function does not appear to be a valid loss function.
